pages/api/user/password/forgotpassword.js

```diff
@@ -1,78 +1,80 @@
 import { hashedPassword } from "../../../../lib/auth";
 import { mailService, mailTemplate } from "../../../../lib/emailservice";
 import { RequestHandler } from "../../../../lib/RequestHandler";
+
 const BASE_URL = process.env.NEXT_APP_URL;
+const SMTP_USER = process.env.SMTP_USER;
 
 async function handle(req, res, prisma) {
-  const reqBody = req.body;
+  const { email } = req.body;
 
-  let alreadyData = await prisma.passwordReset.findUnique({
+  const alreadyResetData = await prisma.passwordReset.findUnique({
     where: {
-      email_id: reqBody.email,
+      email_id: email,
     },
   });
-  let userData = await prisma.user.findUnique({
+
+  if (alreadyResetData)
```

Comment:
We can use exception based system for such error handing .. Like throwing exception in such cases and  use a one hadnler to return a message, so we dont have to write response return logic always